clippy_lints/src/methods/option_map_unwrap_or.rs

```diff
@@ -39,8 +42,8 @@ pub(super) fn lint<'a, 'tcx>(
             }
         }
 
-        // get snippets for args to map() and unwrap_or()
-        let map_snippet = snippet(cx, map_args[1].span, "..");
+        let map_arg_span = map_args[1].span;
```

Comment:
We should move this calculation into the `|db| ..` closure below. It isn't needed otherwise.